File: LeetCode/Easy/0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    x=String(x);
  let y = x.split('').reverse().join('');
  if(x==y){
    return true
  }
  else {
    return false
  }
};

File: LeetCode/Easy/0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if (x < 0 || x != 0 && x % 10 == 0)
            return false;
        int rh = 0, n = x;
        while (rh < n) {
            rh = rh * 10 + n % 10;
            n /= 10;
        }
        return n == rh || n == rh / 10;
    }
};